source/api/cypress-api/env.md

```diff
@@ -121,13 +116,46 @@ Cypress.env("host") // => http://server.dev.local
 
 ```javascript
 Cypress.env({
-  host: "http://server.dev.local",
-  foo: "foo"
+  host: 'http://server.dev.local',
+  foo: 'foo'
 })
 
 Cypress.env() // => {foo: "foo", baz: "quux", host: "http://server.dev.local"}
 ```
 
+## From a plugin
+
+Here's an example that uses `Cypress.env` to access an environment variable that's been {% url 'dynamically set in a plugin' environment-variables#Option-5-Plugins %}.
+
+Use this approach to grab the value of an environment variable _once_ before any of the tests in your fixture run.
```

Comment:
'fixture run', is this meant to say 'spec run'? I don't see the use of fixtures in the example

Reply:
Ohhh... _:: head sinks until it's pressed against my desk ::_

Yes. I failed to context switch: been doing Java of late and specs there are called fixtures. Blah. Lemme change it.

Thanks for your continued attention to detail Jennifer 👍 🙇